openedx/core/djangoapps/content/course_overviews/signals.py

```diff
@@ -26,6 +26,8 @@
 IMPORT_COURSE_DETAILS = Signal()
 # providing_args=["courserun_key"]
 DELETE_COURSE_DETAILS = Signal()
+# providing_args=["courserun_key", "old_name", "new_name"]
+COURSE_NAME_CHANGED = Signal()
```

Comment:
Hmm.. I think we're supposed to put new signals in https://github.com/openedx/openedx-events so they can be used outside of the platform? But no worries if this is beyond the scope of this PR.

Reply:
Not really sure if it has any use case to be included in openedx-events. I had to create it to avoid lms-cms import issues :sweat_smile: